pulsar-broker/src/test/java/org/apache/pulsar/broker/resourcegroup/RGUsageMTAggrWaitForAllMsgsTest.java

```diff
@@ -84,6 +84,7 @@ public long computeLocalQuota(long confUsage, long myUsage, long[] allUsages) {
         this.rgservice = new ResourceGroupService(pulsar, TimeUnit.SECONDS, transportMgr, dummyQuotaCalc);
 
         this.prepareRGs();
+        Thread.sleep(2000);
```

Comment:
Can we use Awaitibility to check if the namespace has been registered? So that we can make the test more stable, 2 seconds might also not be stable enough.

Reply:
@codelipenghui Although the Thread.sleep isn't optimal, I'd like to merge this PR since the test is extremely flaky. @liudezhi2098 please attempt the suggested improvement in a new PR.